app/code/Magento/Elasticsearch/Model/DataProvider/Base/Suggestions.php

```diff
@@ -111,10 +127,13 @@ public function __construct(
         $this->config = $config;
         $this->searchIndexNameResolver = $searchIndexNameResolver;
         $this->storeManager = $storeManager;
-        $this->fieldProvider = $fieldProvider;
```

Comment:
@kuafucode could you re-add this line that's needed for backward compatibility? I would add it myself but I can't edit existing PRs.

I think, at that point, we could merge it quickly ;-)

Reply:
PR is supplemented in https://github.com/mage-os/mageos-magento2/pull/104 , this will be addressed there.